stdlib/2.7/calendar.pyi

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Iterable, Optional, Tuple
+import datetime
+
+YearType = int
+MonthType = int
+DayType = int
+WeekdayType = int
```

Comment:
Honestly I don't think these are very useful -- type checkers don't differentiate between them and the way it's written here these names are exported by the stub (but not by the real module). They just lengthen the signatures.